.circleci/config.yml

```diff
@@ -3,128 +3,137 @@ version: 2.1
 orbs:
   win: circleci/windows@2.2.0
 
-jobs:
-
-  #
-  # linux
-  #
+environment: &global-environment
+  CIBW_BUILD_VERBOSITY: 1
+  CIBW_SKIP: pp* *musllinux*
+  PIP_PROGRESS_BAR: 'off'
 
-  build-manylinux:
+jobs:
+  build-linux:
     parameters:
-      python-minor:
-        type: integer
-      architecture:
+      python-version:
         type: string
 
     docker:
-      - image: quay.io/pypa/manylinux1_<< parameters.architecture >>
+      - image: circleci/python:3.9
 
-    working_directory: /dwave/preprocessing
+    environment:
+      <<: *global-environment
+      CIBW_PROJECT_REQUIRES_PYTHON: ~=<< parameters.python-version>>
+      CIBW_ARCHS_LINUX: x86_64
 
     steps:
       - checkout
-
-      - run:
-          name: fix git checkout
-          command: git reset --hard
-
-      - run:
+      - setup_remote_docker
+      - restore_cache: &build-linux-restore-cache
+          keys:
+            - pip-{{ .Environment.CIRCLE_JOB }}-{{ checksum "pyproject.toml" }}
+      - when:
+          condition:
+            equal: [3.6.8, << parameters.python-version>>]
+          steps:
+            - run: echo 'export CIBW_MANYLINUX_X86_64_IMAGE=manylinux1' >> $BASH_ENV
+      - run: &build-linux-wheels
           name: build wheels
           command: |
-            for PYBIN in /opt/python/*/bin; do
-              if "${PYBIN}/python" -c "import sys; sys.exit(sys.version_info.major == 3)"; then continue; fi;
-              if "${PYBIN}/python" -c "import sys; sys.exit(sys.version_info.minor == << parameters.python-minor >>)"; then continue; fi;
-              "${PYBIN}/pip" install -r requirements.txt --only-binary=:all:
-              "${PYBIN}/pip" wheel . -w ./wheelhouse
-            done
-
-      - run:
-          name: bundle shared libraries into wheels
-          command: |
-            for whl in ./wheelhouse/dwave_preprocessing*.whl; do
-              auditwheel repair "$whl" -w ./dist
-            done
-
-      - store_artifacts:
+            python3 -m venv env
```

Comment:
Do you need to use `python3` here or is just `python` okay?

Reply:
need python3 unfortunately